xkill@eson57/files/xkill@eson57/xkill@eson57.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+if [ ! -e ~/.killdone ]
```

Comment:
It might be better to put this in the directory for the Action itself: `.local/share/nemo/actions/xkill@eson57/.killdone`

Reply:
I can do it, but I don't understand why that would be better. Can you elaborate?

Reply:
This file is only needed and used by this Action. In the event the Action is removed, this file will be left on the user's machine. That's not a good practice. All of the files needed by the Action should be within the proper scope of the Action.

Reply:
You are right, will be done.

Reply:
Done!